Dockerfile

```diff
@@ -0,0 +1 @@
+Dockerfile.fedora30
```

Comment:
You are including only ```Dockerfile.fedora30``` but bellow you are creating a ```Dockerfile.fedora29```, the one expected to use is Fedora 30 ?

Reply:
Oops, path is wrong. It should be `fedora/Dockerfile` istead of `Dockerfile`. 
It's a symlink pointing to the latest one. But, yes, I'm adding 29 just in case.

Reply:
Another observation: Also, there is a ```Dockerfile``` within  fedora directory

Reply:
K, thanks for force push.

The build of ```fedora30``` brokes with dependency hell in dnf:

``` bash
Error:
 Problem: problem with installed package fedora-release-container-30-1.noarch
  - package fedora-release-container-30-1.noarch conflicts with system-release provided by fedora-release-30-1.noarch
  - package fedora-release-30-1.noarch conflicts with system-release provided by fedora-release-container-30-1.noarch
  - package fedora-release-container-30-3.noarch conflicts with system-release provided by fedora-release-30-1.noarch
  - package fedora-release-30-1.noarch conflicts with system-release provided by fedora-release-container-30-3.noarch
  - conflicting requests
  - package fedora-release-container-30-1.noarch conflicts with system-release provided by fedora-release-30-3.noarch
  - package fedora-release-30-3.noarch conflicts with system-release provided by fedora-release-container-30-1.noarch
  - package fedora-release-30-3.noarch conflicts with system-release provided by fedora-release-container-30-3.noarch
  - package fedora-release-container-30-3.noarch conflicts with system-release provided by fedora-release-30-3.noarch
```